docker-compose.yml

```diff
@@ -356,10 +356,10 @@ services:
       - _APP_EXECUTOR_SECRET
       - _APP_EXECUTOR_HOST
 
-  appwrite-worker-databases:
-    entrypoint: worker-databases
+  appwrite-worker-databases-01:
+    entrypoint: worker-databases 01
     <<: *x-logging
-    container_name: appwrite-worker-databases
+    container_name: appwrite-worker-databases-01
```

Comment:
Do we need the number postfix? Would be more elegant if we can skip on the default self-hosted setup. When needing to scale, like in Cloud, devs can have a dedicated and custom compose file.